### ISSUE TITLE
Add Basic Python Scripts

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/Awesome_Python_Scripts#1]
This issue is specfically created for those who are new to python and open source. 

Repository : Basic Python Scripts (Beginner friendly)

- Do check if the code you want to contribute is available in this repository
- Comment on this issue with the title and details of the script you are going to contribute and what does it do
- Start working on that script and before you add make sure it works properly
- Add your code to 'Basic Python Scripts' repository using the contribution guidelines and template mentioned 
- Create a Pull Request and be patient
- I'll review your code and if all works perfectly fine, PR will be merged.
- If anything else needed from you, I'll get back with the response

Any queries, don't hesitate to reach out to me. I'll be happy to support and mentor you.

Happy Contribution!

**Comment by WebDesign4New** [comment]
Hello mam. I want to contribute to this. Shall I begin with basic python programs?

**Comment by prathimacode-hub** [comment]
Yes, you can. You don't have to ask. Hope you have seen the contributing guidelines and template to follow. Mention the Python script you are contributing and what it does here. That's all it needs. And you are ready to go. @WebDesign4New

**Comment by sonaliBedade** [comment]
Python script name: Jumping number  (Takes input from the user and determines whether the number entered is a Jumping number or not). Ma'am can I start working on this one?

**Comment by prathimacode-hub** [comment]
Yes, Go ahead @sonaliBedade

**Comment by Krishna841** [comment]
Hey, @prathimacode-hub !!! I wish to contribute. Is there any specific project you would like?

**Comment by prathimacode-hub** [comment]
That's a wonderful initiative. Sure, you're free to contribute. Anything that's useful for beginners and as well that would serve some purpose out of it. @Krishna841

**Comment by Krishna841** [comment]
Okay, can I add the script to do a binary search?(it will perform binary search on either randomly generated array or through user input array)

**Comment by prathimacode-hub** [comment]
That wouldn't justify this issue, as it is on algorithms and data structures. I will create issue on it in near future. But for now do prefer basic programs like calculator, alarmclock, temperatureconverter, urlshortener, typingspeed, musicplayer, screenshot taker, password generator, dictionary, ascii art generator, youtube downloader, portscanner etc. @Krishna841

**Comment by prathimacode-hub** [comment]
Sure @Krishna841

**Comment by Krishna841** [comment]
Hey, @prathimacode-hub !!! I'll be making a chatbot that can do some basic conversations.
Sounds good?

**Comment by prathimacode-hub** [comment]
That sounds awesome. It's a wonderful project. Go for it. 👍
Once you're done, share me the link so that I can think of on where it can be placed. Based on that, I will suggest the repository and you can create a PR. 
There are number of ways chatbot can be implemented. Are you using any GUI or automating the chatbot? @Krishna841

**Comment by Krishna841** [comment]
Umm... mostly I'm thinking of using python libraries. This is just a sample which would be the most basic one.

**Comment by prathimacode-hub** [comment]
Ok, cool. All the best. Can't wait to see ur program working. @Krishna841

**Comment by neelshah2409** [comment]
Mam I had made password generator and typing speed test ,may I contribute it here. And please guide me for that.

**Comment by prathimacode-hub** [comment]
Sure, you can. Please go ahead. First raise a separate issue for each project you are going to contribute. Before you start, go through README of the project and contributing guidelines given for smooth flow of PR. Once you raise an issue, I shall assign the project to you and would provide a guidance as well. @neelshah2409

**Comment by pratimakush10** [comment]
Ma'am I want to contribute in *Basic Python Scripts #1*
I had made simple alarm clock by importing datetime module please guide me as its my first open source contribution

**Comment by prathimacode-hub** [comment]
Sure, I shall guide you. Tomorrow, after 10AM raise an issue using the template given in README and get it asigned to you. Are you aware of Github workflow? @pratimakush10

**Comment by pratimakush10** [comment]
yes mam ,I know the BASIC Github workflow of git fork and git clone.

**Comment by prathimacode-hub** [comment]
That's cool. The main thing being follow the README and CONTRIBUTING GUIDELINES mentioned and you are good to go. Raise a new issue of the project you are interested to work on after the stipulated time tomorrow using the template. I shall guide you. Any queries, don't hesitate to ask. @pratimakush10

**Comment by pratimakush10** [comment]
yes mam, I will follow the  README and CONTRIBUTING GUIDELINES and surely I will ask query  if faces any problem.

**Comment by sumitsoni0907** [comment]
Hello mam I want to contribute in bsic python script. please give the idea which type of script i want to contribute.

**Comment by prathimacode-hub** [comment]
Select the script you're comfortable working on. There are wide range of repositories. Choose any of it and come up with an idea. If you need specific idea, you would know better yourself. Be unique and creative. Also make sure, the script is not available in the repo. @sumitsoni0907

**Comment by sumitsoni0907** [comment]
I am working on string anagram or not. anagram string means second string is rearrangement of first string like 'race' and 'care'.
It is valid for contributing. its my first contribution.
I have another idea number is Peterson or not. Peterson number means sum of factorials of each digit is equal to number itself like 145 = 1!+4!+5!=1+24+120=145.
give me the answer both valid or not for contributing.

**Comment by prathimacode-hub** [comment]
Create a new issue using the template given and get it assigned. @sumitsoni0907

**Comment by sumitsoni0907** [comment]
hello mam I create new issue for anagram string please assign me

**Comment by sumitsoni0907** [comment]
> I have another idea number is Peterson or not. Peterson number means sum of factorials of each digit is equal to number itself like 145 = 1!+4!+5!=1+24+120=145.
I will create new script for peterson number. Is it valid?

**Comment by prathimacode-hub** [comment]
Yes, its valid. Once you complete earlier issue, you can create a new issue for this script. @sumitsoni0907

**Comment by nirbhay12345** [comment]
I wish to contribute. does a 'image to text' convertor meets the requirements of the repo?

**Comment by prathimacode-hub** [comment]
Yes, sure. You are free to do so. First create an issue using issue template and express your interest to get it assigned. @nirbhay12345

**Comment by nirbhay12345** [comment]
Okay Thank you @prathimacode-hub .
